extension/EdFiLMSMetaEd/package.json

```diff
@@ -1,6 +1,11 @@
 {
+  "name": "ed-fi-lms-1",
+  "version": "1.0.0",
+  "description": "The Ed-Fi LMS extension data model",
+  "repository": "https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit",
+  "license": "Apache-2.0",
   "metaEdProject": {
-    "projectName": "EdFiLMS",
+    "projectName": "Ed-Fi-LMS",
```

Comment:
The old project name caused strange problems. Somewhere in MetaEd or in Code Generation, it was being translated into `ed-fi-lms` - but not consistently. In the XSD retriever class in the bulk load process in particular, I could see failures due to these two strings not matching.

These screenshots are from before the change. After the change, uploading descriptors worked just fine.

![image](https://user-images.githubusercontent.com/9324390/122655591-07316f80-d119-11eb-95a2-ce53f912610d.png)

![image](https://user-images.githubusercontent.com/9324390/122655593-0e587d80-d119-11eb-934d-6c43fe97884b.png)